crux-llvm/src/Crux/LLVM/Log.hs

```diff
@@ -25,6 +25,7 @@ data CruxLLVMLogMessage
   | FailedToBuildCounterexampleExecutable
   | SimulatingFunction Text
   | UsingPointerWidthForFile Integer Text
+  | TranslationWarning Text
```

Comment:
Rather than stuff the entire warning message here as a bare `Text`, it would probably be more in the spirit of structured message logging to store the list of warning triples (or the newtypes on top of them here), and then migrate the pretty-printing code from `sayTranslationWarning` to `cruxLLVMLogMessageToSayWhat`.